pingcap/tidb
# planner: binding from cluster statements summary

### What problem does this PR solve?


Issue Number: ref #39199

Problem Summary:

### What is changed and how it works?
Support qiuick binding from cluster statements summary table
### Check List

Tests 


Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### Manual test
use tiup playground start a cluster with 2 tidb node, execute a sql in one node

<img width="1512" alt="截屏2022-12-13 上午11 32 23" src="https://user-images.githubusercontent.com/108643977/207220186-4e19cf1e-135a-4aa9-846f-eca9ea57bf51.png">

and create binding and verify it works in `session | global` scope in another node
<img width="1512" alt="截屏2022-12-13 上午11 38 02" src="https://user-images.githubusercontent.com/108643977/207220935-b4650437-fd03-4d99-8f82-57683bea7280.png">

<img width="1512" alt="截屏2022-12-13 上午11 39 08" src="https://user-images.githubusercontent.com/108643977/207220944-52bf515b-80f6-4c1f-b49a-6ad317f058f5.png">
